robotframework-ls/package.json

```diff
@@ -441,6 +441,11 @@
                     "default": "",
                     "description": "Load variables for code-completion and code-analysis based on an arguments file. Multiple files accepted by separating with a comma."
                 },
+		"robot.loadVariablesFromVariablesFile": {
```

Comment:
There should be at least one test-case for this setting.

See robotframework_ls_tests.test_vscode_robot.test_variables_completions_integrated_using_config as a base.